crates/wkg/src/main.rs

```diff
@@ -199,6 +199,13 @@ struct GetArgs {
     #[arg(long, value_enum, default_value = "auto")]
     format: Format,
 
+    /// Check that the got package matches the existing file at the output
```

Comment:
Spelling/grammar (normally these are nits, but since it is help text, these should be fixed): "fetched" or "retrieved" instead of "got" and "simmilar" -> "similar"

Reply:
Fixed